src/agent/block_store_services/block_store_azure.js

```diff
@@ -53,6 +53,17 @@ class BlockStoreAzure extends BlockStoreBase {
         return this.blob.generateSharedAccessSignature(this.container_name, block_key, shared_access_policy);
     }
 
+    _get_block_store_info() {
+        const connection_params = {
+            connection_string: this.cloud_info.azure.connection_string,
+        };
+        return {
+            connection_params,
+            target_bucket: this.cloud_info.target_bucket,
+            blocks_path: this.blocks_path,
+        };
+    }
+
     _delegate_read_block(block_md) {
```

Comment:
do we still need the old `delegate_read_block` and `delegate_write_block`?

Reply:
removed the old code